client/src/components/Banner.jsx

```diff
@@ -1,5 +1,7 @@
 import React from 'react';
 
-const Banner = () => <div>Logo</div>;
+const Banner = () => (
+  <header>Project Catwalk</header>
```

Comment:
I think you should place this in the index.html file due to the semantic meaning of the header element. If you decide that then remove the Banner component. then on line 8 in index.html have `<header><h1>Project Catwalk</h1></header>`

If you still want to use a Banner Component, then use an h1 element instead of a header.

Reply:
The definition of the header element that I found said it is meant to contain things like a log and nav elements, so I feel like header would be appropriate here.

Reply:
My thought was to have header as well due to the fact that it had a logo and could potentially get a search feature if built.

Reply:
You are both correct. I agree a header is more appropriate here